### DISCUSSION TITLE
Use beyond the desktop

### DISCUSSION THREAD

**Discussion by alsoeric** [borgbase/vorta#1422] [General]
have done an amazingly wonderful job with vorta. It is the kind of dashboard I would want for Borg even in a server context.  From my perspective, vorta handles almost all the cases I would need for backup. The only thing that seems to be missing is sending out an alert if something fails

Two questions: 
is there any reason why you would not use vorta in a server context?
How can I sent an alert when there is a borg level error?

**Comment by m3nu** [comment]
You probably want to use Borgmatic in a server context.

If you do want something graphical, there is also Vorta-Docker https://github.com/borgbase/vorta-docker
